libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -1717,6 +1717,16 @@ Compass::use_for_yaw(uint8_t i) const
     return _use_for_yaw[Priority(i)] && _learn.get() != LEARN_INFLIGHT;
 }
 
+uint8_t Compass::get_first_usable() const
```

Comment:
add a comment please, explaining relationship to COMPASS_USE

Reply:
No longer applicable